dockerfiles/theia/versions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+set -e
+
+THEIA_VERSION="0.3.10"
```

Comment:
Shouldn't we use `${THEIA_VERSION}` here since such an env is set with build arg?